subsys/bluetooth/audio/bap_internal.h

```diff
@@ -0,0 +1,93 @@
+/** @file
+ *  @brief Internal header for Bluetooth BAP.
+ */
+
+/*
+ * Copyright (c) 2019 Bose Corporation
+ * Copyright (c) 2021-2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Sometimes the contents of the include file is guarded against double includes with
```
#ifndef ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_BAP_INTERNAL_H_ 
#define ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_BAP_INTERNAL_H_  
... 
#endif
```

Any particular reason for not doing this here?

Reply:
We don't do that for internal header files :) Only public ones AFAIK (I've been asked to remove guards like these previously).